### DISCUSSION TITLE
Multiple gradient brushes in styles?

### DISCUSSION THREAD

**Discussion by KieranDevvs** [AvaloniaUI/Avalonia#6355] [Q&A]
Is it possible to have multiple gradients as a control background, and if so, how?

Ive managed to set a single gradient like so:
```xml
<Setter Property="Background">
    <Setter.Value>
        <LinearGradientBrush StartPoint="50%, 0%" EndPoint="50%, 100%">
            <GradientStops>
                <GradientStop Offset="0" Color="#FFF" />
                <GradientStop Offset="0.5" Color="#AAA" />
                <GradientStop Offset="1" Color="#FFF" />
            </GradientStops>
        </LinearGradientBrush>
    </Setter.Value>
</Setter>
```

**Comment by maxkatz6** [comment]
What do you mean by "multiple gradients"? Multiple layers of different semi-opaque gradients?
Is possible, but not that easy as just set Background.
1. You can override template and put multiple Border controls (layering them) with different Background values. 
2. I think it's also possible with VisualBrush, where you put multiple layers of Borders inside of it, and then just use as value of Background property. A bit more complex approach, but does not requires changing default template.

**Reply by KieranDevvs** [reply]
Thanks, i'll give that a try. (To answer your question, yeah I meant layered. The end goal is to blend them)
